edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/EdgeHubDesiredProperties.cs

```diff
@@ -55,17 +70,33 @@ internal void ValidateSchemaVersion()
                         throw new InvalidSchemaVersionException($"Route priority/TTL is not supported in schema {this.SchemaVersion}.");
                     }
                 }
-            }
+
+                // Authorization policies not allowed.
+                if (this.BrokerConfiguration != null)
+                {
+                    throw new InvalidSchemaVersionException($"Broker configuration is not supported in schema {this.SchemaVersion}.");

```

Comment:
I'm removing strict schema validation as per an earlier design decision change, so you're probably going to see a conflict here as soon as you merge.
[https://github.com/Azure/iotedge/pull/3698](https://github.com/Azure/iotedge/pull/3698)